src/styling/ProjectsPage.css

```diff
@@ -1,101 +1,174 @@
 /** Multiple card view **/
 
-.pg-container {
+.projects-page {
   max-width: 1400px;
+  margin: 3rem auto;
+}
+
+.projects-page .flow-images {
+  margin: 1rem 0;
 }
 
 .preview-card {
-  border-radius: 10px;
-  margin: 20px 0px;
-  padding: 5px;
+  border-radius: 16px;
+  margin: 16px 0px;
+  box-shadow: 8px 8px 0px 2px rgba(249,245,227,1);
```

Comment:
I forget, did we want to use cream for the border and text? @Raquellevy for confirmation.